test/unit/model/v2/test_platforms_section.py

```diff
@@ -105,6 +107,7 @@ def _model_platforms_docker_errors_section_data():
             'name': int(),
             'hostname': int(),
             'image': int(),
+						'pull': str(),
```

Comment:
Can you change this to `'pull': init()`, this will force the validation to error, which down below the assert will need updating.

Reply:
Now deliberately failing using `'pull': int(),`